apps/codeforafrica/src/components/OpportunityCard/OpportunityCard.js

```diff
@@ -0,0 +1,60 @@
+import { RichTypography } from "@commons-ui/core";
+import { Link } from "@commons-ui/next";
+import {
+  Card,
+  CardContent,
+  CardActions,
+  CardMedia,
+  Button,
+  Box,
+} from "@mui/material";
+import React from "react";
+
+const OpportunityCard = React.forwardRef(function OpportunityCard(props, ref) {
+  const { title, description, image, date, tags, comments, href } = props;
+
+  if (!title || !description) {
+    return null;
+  }
+  return (
+    <Card sx={{ mt: "80px", boxShadow: "none", borderRadius: 0 }} ref={ref}>
```

Comment:
Can't see the need to have margins affecting _outside_ the component. If needed, they can be set in the component using this card.

Reply:
Made a change, but could you paraphrase this comment? @kilemensi

Reply:
Sure @gertie-sheshe: A component shouldn't deal with what's happening outside its boundaries i.e. `margin-top` on a `Card` in this instance. Let another component that uses this component decide what should happen _outside_ the child component. e.g. A `CardList` can set `margin-top` on `Card`s because while `margin-top` is outside a single `Card`, it will still be inside the `CardList,` etc.